src/cast_core/CMakeLists.txt

```diff
@@ -14,46 +14,10 @@
 
 project(cast_core VERSION 0.0.1 LANGUAGES CXX)
 
-# Pacify the CLion
-file(GLOB_RECURSE HEADERS_FOR_CLION
-     RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
-     CONFIGURE_DEPENDS
-     include/*.hpp include/*.hh src/*.hh)
-add_library(cast_core_headers SHARED
-    EXCLUDE_FROM_ALL
-    src/cast_core.cpp
-    ${HEADERS_FOR_CLION}
-)
-
-add_library(cast_core SHARED
-    # ActorsStart
-    src/actors/HelloWorld.cpp
-    src/actors/Insert.cpp
-    src/actors/InsertRemove.cpp
-    src/actors/Loader.cpp
-    src/actors/MultiCollectionQuery.cpp
-    src/actors/MultiCollectionUpdate.cpp
-    src/actors/RunCommand.cpp
-    # ActorsEnd
```

Comment:
we glob all cpps now so this isn't needed and removed the grossness in `create-new-actor.sh` to update this file 🎉